src/pages/experiments/[experimentId]/data-processing/data-integration/components/DataIntegration.jsx

```diff
@@ -30,36 +48,153 @@ const defaultStylingConfig = {
+  toggleInvert: '#FFFFFF',
 };
 
+const { Panel } = Collapse;
+
 const DataIntegration = () => {
-  const [plotConfig, setPlotConfig] = useState(defaultStylingConfig);
+  const router = useRouter();
+  const { experimentId } = router.query;
+
+  const [selectedPlot, setSelectedPlot] = useState('sample');
+
+  const [config, setConfig] = useState(defaultStylingConfig);
 
-  const updatePlotWithChanges = (plotConfigUpdates) => {
-    const newPlotConfig = _.cloneDeep(plotConfig);
-    _.merge(newPlotConfig, plotConfigUpdates);
+  const updatePlotWithChanges = (configUpdates) => {
+    const newConfig = _.cloneDeep(config);
+    _.merge(newConfig, configUpdates);
 
-    setPlotConfig(newPlotConfig);
+    setConfig(newConfig);
+  };
+
+  const plots = {
+    sample: {
+      title: 'Samples',
+      imgSrc: cellBySamplePic,
+    },
+    frequency: {
+      title: 'Default clusters',
+      imgSrc: frequencyPic,
+    },
+    elbow: {
+      title: 'Mitochondrial fraction reads',
+      imgSrc: elbowPic,
+    },
   };
 
   return (
     <>
       <Row>
-        <Col span={16}>
-          <ElbowPlot plotConfig={plotConfig} />
+        <Col span={14}>
+          <ElbowPlot config={config} />
         </Col>
-        <Col span={1}>
-          <Tooltip title='Dimensionality reduction is necessary to summarise and visualise single cell data. The most common method is Principal Component Analysis (PCA). The user sets the maximum number of PCs.'>
-            <Button icon={<InfoCircleOutlined />} />
-          </Tooltip>
+
+        <Col span={3}>
+          <Space direction='vertical'>
+            <Tooltip title='The number of dimensions used to configure the embedding is set here. This dictates the number of clusters in the Uniform Manifold Approximation and Projection (UMAP) which is taken forward to the ‘data exploration’ page.'>
+              <Button icon={<InfoCircleOutlined />} />
+            </Tooltip>
+
+            {Object.entries(plots).map(([key, plot]) => (
+              <button
+                type='button'
+                key={key}
+                onClick={() => setSelectedPlot(key)}
+                style={{
+                  padding: 0, margin: 0, border: 0, backgroundColor: 'transparent',
+                }}
+              >
+                <img
+                  alt={plot.title}
+                  src={plot.imgSrc}
+                  style={{
+                    height: '100px',
+                    width: '100px',
+                    align: 'center',
+                    padding: '8px',
+                    border: '1px solid #000',
+                  }}
+                />
+              </button>
+
+            ))}
+          </Space>
         </Col>
+
         <Col span={7}>
-          <CalculationConfig />
-          <PlotStyling
-            config={plotConfig}
-            onUpdate={updatePlotWithChanges}
-            singlePlot
-          />
+          <Collapse defaultActiveKey={['data-integration']}>
```

Comment:
Could you move the styling elements over to a different component? Something that can replace `PlotStyling` perhaps

Reply:
Yea, this was something I talked about with @ivababukova  for #95 . This will be investigated and looked into.

There are a lot of redundant duplication of codes. However, as the PR shows, refactoring something that affects a large part of the codebase can be complicated, especially with multiple merge requests being done at the same time (merge conflicts).

Reply:
yup, we have a ticket to clean these things up for next sprint